MatrixKit/Models/Room/MXKAttachment.h

```diff
@@ -0,0 +1,79 @@
+/*
+ Copyright 2015 OpenMarket Ltd
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+#import <MatrixSDK/MatrixSDK.h>
+
+/**
+ List attachment types
+ */
+typedef enum : NSUInteger {
+    MXKAttachmentTypeUndefined,
```

Comment:
Does it worth to keep MXKAttachmentTypeUndefined?
It was not used before this refactoring and it is still unused after.